kotlin/android/app/src/main/java/dev/firezone/android/features/session/ui/ViewResource.kt

```diff
@@ -4,10 +4,12 @@ package dev.firezone.android.features.session.ui
 import dev.firezone.android.tunnel.model.Resource
 import dev.firezone.android.tunnel.model.Site
 import dev.firezone.android.tunnel.model.StatusEnum
+import dev.firezone.android.tunnel.model.TypeEnum
```

Comment:
I know you are just following what is already been done but data-types in names is really odd. This should just be called `ResourceType` IMO.